Dockerfile

```diff
@@ -16,12 +26,14 @@ ARG PROJECT=akash
 ARG PROJECT_BIN=$PROJECT
 ARG VERSION=v0.12.1
 ARG REPOSITORY=https://github.com/ovrclk/akash.git
+ARG MAKE_ENV
+ENV MAKE_ENV=$MAKE_ENV
 
 # Clone and build project
 RUN git clone $REPOSITORY /data
 WORKDIR /data
 RUN git checkout $VERSION
-RUN make install
+RUN export $MAKE_ENV; make install
```

Comment:
Would this export error if `MAKE_ENV` wasn't set?

Reply:
running just ```export``` will just print the existing environment variables, but will not give any errors. I can give it a default valuet to MAKE_ENV if we don't want the existing variables to be printed.